web/src/filter.ts

```diff
@@ -665,7 +665,7 @@ export class Filter {
 
             // Note: We hack around using this in "describe" below.
             case "has":
-                return verb + "messages with one or more";
+                return verb + "messages with";
 
```

Comment:
Do you understand what the "We hack around ..." comment there is about? It seems important.

Reply:
I guess that was added in 1b07b32ec1b6e66c18fb8f71fe33649294dcb11a; I suspect we just forgot to clean up that comment.